src/fides/api/models/privacy_experience.py

```diff
@@ -503,7 +555,10 @@ def remove_config_from_matched_experiences(
     ).all()
 
     for experience in experiences_to_unlink:
-        experience.unlink_experience_config(db)
+        if experience_config.is_default:
+            experience.unlink_experience_config(db)
+        else:
+            experience.link_default_experience_config(db)
```

Comment:
This is the other secondary change of this PR.  If a user updates an experience config to remove a region (not currently exposed in the latest UI but will be a future iteration), I link to the default experience config in the interim.